core/services/relay/evm/cap_encoder.go

```diff
@@ -102,14 +102,20 @@ func extractIDs(input map[string]any) ([]byte, []byte, []byte, []byte, error) {
 		return nil, nil, nil, nil, err
 	}
 
-	// TODO: source donID and workflowOwner from somewhere
-	donID := []byte{0, 1, 2, 3}
-	workflowOwner := make([]byte, 32)
+	donID, err := decodeID(input, consensustypes.DonIDFieldName, 4)
```

Comment:
@bolekk @DeividasK did we decide on the typing here? Is it a 4 byte id? Are we going to use `string` for readability?